src/main/scala/org/broadinstitute/dsde/firecloud/webservice/WorkspaceApiService.scala

```diff
@@ -107,8 +108,16 @@ trait WorkspaceApiService extends HttpService with FireCloudRequestBuilding
             }
           } ~
           path("updateAttributes") {
-            requireUserInfo() { _ =>
-              passthrough(workspacePath, HttpMethods.PATCH)
+            patch {
+              requireUserInfo() { userInfo: UserInfo =>
+                entity(as[Seq[AttributeUpdateOperation]]) { replacementAttributes => requestContext =>
+                  perRequest(requestContext,
```

Comment:
This changed from `passthrough` to `perRequest` to accommodate the `republishIfPublished` call